testing/notebooks/generate_matrix-benchmarking.sh

```diff
@@ -132,20 +133,21 @@ generate_matbench::generate_visualization() {
 
     cp -f /tmp/prometheus.yml "." || true
     if ! matbench parse |& tee > "$ARTIFACT_DIR/_matbench_parse.log"; then
-        echo "An error happened during the parsing of the results (or no results were available) in $ARTIFACT_DIR, aborting."
+        _warning "An error happened during the parsing of the results (or no results were available) in $MATBENCH_RESULTS_DIRNAME, aborting."
         return 1
     fi
 
-    if ! matbench parse --output_lts $ARTIFACT_DIR/lts_payload.json |& tee > "$ARTIFACT_DIR/_matbench_parse_lts.log"; then
-        echo "An error happened while encoding results into a JSON object within $ARTIFACT_DIR, aborting."
-        return 1
+    retcode=0
+
+    if ! matbench parse --output_lts $ARTIFACT_DIR/lts_payload.json |& tee > "$ARTIFACT_DIR/_matbench_generate_lts.log"; then
+        _warning "An error happened while generating the LTS payload from $MATBENCH_RESULTS_DIRNAME :/."
+        retcode=1
```

Comment:
here, it's important not failing: the test ran, maybe it took hours to execute, we don't want to abort the visualization because the LTS generation failed

(in the block above, if the parsing fails, well, the plot generation has no chance to succeed, that's why it aborts)

and also, I don't swallow the error, but with the `_warning` command I [create a file in the artifacts](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/693/pull-ci-openshift-psap-ci-artifacts-main-ods-notebooks-light/1650811533887803392/artifacts/notebooks-light/test/artifacts/) that makes it clear that something wrong happened. Actually, the test will be marked as `failed` thanks to `retcode=1`, but the execution won't be aborted